test/cases/expressions/xhp-class.hack

```diff
@@ -1,4 +1,4 @@
-class :a:m-b {
+xhp class a:m_b {
 }
 
-new :a:m-b:b-m();
+new a:m_b:b_m();
```

Comment:
Since we're going to be supporting both v3 and v4 can we keep this test as is ad rename it `xhp-class-v3.hack` and create a new test file `xhp-class-v4.hack` with the new syntax?

Reply:
Yes, that's a good idea - I'll make those changes.